public/app/percona/dbaas/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useMemo } from 'react';
+import { Field } from 'react-final-form';
+import { cx } from 'emotion';
+import { Icon, Tooltip, Switch, useStyles } from '@grafana/ui';
+import { compose } from '@percona/platform-core/dist/shared/validators';
+import { SwitchFieldProps } from './Switch.types';
+import { getStyles } from './Switch.styles';
+import { SwitchFieldRenderProps } from './Switch.types';
+
+export const SwitchField: FC<SwitchFieldProps> = ({
```

Comment:
Why not adding this component to `platform-core`?

Reply:
Will add it after the upgrade.